app/models/MetricsData.scala

```diff
@@ -0,0 +1,31 @@
+package models
+
+sealed trait PublishingSystem
+case object Composer extends PublishingSystem
+case object InCopy extends PublishingSystem
+
+case class Metrics (
+    id: String,
+    contentCreated: Long,
+    composerId: Option[String],
+    storyBundleId: Option[String],
+    firstSeenInWorkflow: Option[Long],
```

Comment:
This is the model for deserialising/serialising for DynamoDB. DynamoDB has a very limited number of datatypes it can handle :(